.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Could you please avoid versioning configuration files specific to your [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)?

FLOSS developpers use many different tools to write code, and it would be preferable to keep IDE-specific configurations files unversioned.

You can however add the `.idea/` directory to [this repo root `.gitignore`](https://github.com/py-pdf/pdfly/blob/main/.gitignore) 🙂

Reply:
If you meant "directory" instead of "diagram" then yes! 🙂

Else I'm not sure about which diagram you are refering to 😄

Reply:
i mean the picture i showed. hhh